### PR TITLE
Stack trace sometimes does not show line number of the spec where it failed

### PR DESCRIPTION
**Pull request by heraldantony**
update target to es2019 in tsconfig.json to fix [issue](https://github.com/SAP/wdio-qmate-service/issues/60)

Please see webdriver issue - https://github.com/webdriverio/webdriverio/pull/8129

### CODE REVIEW

File: test/reuse/nonUi5/element/getById.spec.js [hunk 1]
```diff
@@ -32,4 +32,21 @@ describe("element - getById and catch error", function () {
...
+describe("element - getById - error case, verify error stack", function () {
+
+  it("Preparation", async function () {
+    await common.navigation.navigateToUrl("https://sapui5.hana.ondemand.com/1.99.0/");
+    await handleCookiesConsent();
+  });
+
+  it("Execution & Verification", async function () {
+    try {
+      await nonUi5.element.getById("sdk---app--apiTab-text");
```
Comment (Templada) [inline test/reuse/nonUi5/element/getById.spec.js:46]: can we change to expectToThrow or sth. similar?

**Review by Templada — CHANGES_REQUESTED** [review]
see comments

File: test/reuse/nonUi5/element/getById.spec.js (hunk 1 above)
Comment (heraldantony) [reply test/reuse/nonUi5/element/getById.spec.js:46]: @Templada - I couldn't find anything that would match the error stack. `rejects.toMatch`, `rejects.toThrow` and related methods only match the error message, not the stack trace.

**Comment by DIFSRIP** [comment]
any open point or ready to merge?

**Review by Templada — APPROVED** [review]

**Review by DIFSRIP — APPROVED** [review]
looks good

**Pull request merged**